pensjon-brevbaker/.nais/nais.yaml

```diff
@@ -40,7 +40,7 @@ spec:
   azure:
     application:
       enabled: true
-      allowAllUsers: {{allowAllUsers}}
+      allowAllUsers: false
```

Comment:
Vet vi hva som skjer når pesys bruker client-credentials token (maskin-til-maskin)?

Reply:
Jeg lurer litt på om vi skal holde brevbaker utenfor groups-claim endringen. Team etterlatte bruker også brevbaker, og jeg vet ikke hvilke grupper deres brukere har.

Reply:
Ja, var litt usikker der... allowAllUsers var jo false tidligere også, men da uten at grupper var spesifisert

Reply:
Hvis det er snakk om maskin-til-maskin tror jeg det kun er styrt av inbound rules, per doc her: https://doc.nais.io/security/auth/azure-ad/configuration/#applications

Reply:
Så tror det skal være riktig. Men teste kun dev først kanskje?

Reply:
Hmm, allowAllUsers var false i prod og true i dev ja. Men vi legger jo opp til at skribenten sender et obo-token for bruker til brevbaker, så da må jo det aksepteres og vi må enten ha allowAllUsers eller gruppe. Siden det fungerer i produksjon for team etterlatte med false, så burde det jo ikke være noe verre om vi legger til en gruppe? Det betyr nok at de bruker client-credentials mot brevbaker.